lib/tarpon/request/subscriber.rb

```diff
@@ -12,6 +12,10 @@ def get_or_create # rubocop:disable Naming/AccessorMethodName
         perform(method: :get, path: path, key: :public)
       end
 
+      def update(**data)
```

Comment:
I'd like to see this method in a separate class dedicated for the `Attribute` resource, to keep it consistent to the standards followed by the library so far, which is to implement a 1:1 mapping to the way RevenueCat designs their REST API. The same concept is present in [offerings](https://github.com/fishbrain/tarpon/blob/master/lib/tarpon/request/subscriber.rb#L27-L29) and [subscriptions](https://github.com/fishbrain/tarpon/blob/master/lib/tarpon/request/subscriber.rb#L31-L33).

This also makes it future-proof in case RevenueCat supports more operations on subscriber attributes.

The API would look like this:
```ruby
Tarpon::Client.subscriber('app_user_id').attributes.update(...)
```

Reply:
@igorbelo ready for another review!